security/advancedtls/testdata/crl/provider_extensions.conf

```diff
@@ -1,4 +1,5 @@
-subjectKeyIdentifier = hash
+[extensions]
+subjectKeyIdentifier = 54:1B:29:00:6E:3C:B1:54:68:75:C2:F8:26:90:BB:46:D5:70:45:E9
```

Comment:
What does setting this do exactly? It just immediately strikes me as a little odd to hard-code a value like this in what is meant as a configuration file?

Reply:
That's a bug, I forgot to include a 'revert' part of the script. The updated flow is
Create a cert
Grab it's subjectKeyIdentifier
Update the value of subjectKeyIdentifier in .cnf
Generate malicious certs using that .cnf
Revert .cnf back to subjectKeyIdentifier = hash